.eslintrc.json

```diff
@@ -30,6 +30,43 @@
     "no-process-exit": 0,
     "require-atomic-updates": 0,
     "no-undef": [2, { "typeof": true }],
+
+    "max-len": [
+      2,
+      {
+        "code": 120,
```

Comment:
[sand] Can we drop line length and tab width and enforce it via prettier (seems like length is already enforced).

Reply:
Right. EditorConfig enforces it too (on top of Prettier), so we can drop this. :+1: